resources/assets/js/app.js

```diff
@@ -17,6 +17,8 @@ require('./bootstrap');
 Vue.component('example', require('./components/Example.vue'));
 Vue.component('TeacherKanban', require('./components/TeacherKanban.vue'));
 Vue.component('FormInput', require('./components/Forms/FormInput.vue'));
+Vue.component('SearchStudent', require('./components/Forms/SearchStudent.vue'));
+Vue.component('TeachersSearch', require('./components/Forms/TeachersSearch.vue'));
```

Comment:
shouldn't this be `SearchTeacher` the same way u called the other `SearchStudent`